.github/workflows/golangci-lint.yaml

```diff
@@ -25,5 +25,21 @@ jobs:
 
           # Optional: golangci-lint command line arguments.
           args: --timeout=4m
+
+          # Optional: show only new issues if it's a pull request. The default value is `false`.
+          # only-new-issues: true
+
+      - name: gosec
+        uses: golangci/golangci-lint-action@v2
+        with:
+          # Required: the version of golangci-lint is required and must be specified without patch version: we always use the latest patch version.
+          version: v1.29
+
+          # Optional: working directory, useful for monorepos
+          # working-directory: somedir
+
+          # Optional: golangci-lint command line arguments.
+          args: --disable-all -E gosec  pkg/... -severity=medium cmd/... -severity=medium
```

Comment:
We need a separate rule since the args are different for gosec? What is the default severity and what if we ran with medium severity on the other rules?

Reply:
for gosec `args: pkg/... -severity=medium && cmd/... -severity=medium` it almost the same and default severity in golangci is `default-severity: error` and I can't see any difference or more/less errors when running with severity or without.

Reply:
Since we get the same errors with the default severity, can we combine the gosec with the other golangci-lint rule? Or why does it need to be separate?

Reply:
Ok if it's just a matter of handling errors in the tests, sounds good to do that so we can combine it with the other rule. And we can suppress the gosec errors more freely in the tests as well to keep it simple.

Reply:
Done.